### ISSUE TITLE
Feature request: 8-bit parallel mode

### ISSUE THREAD

**Issue by Derek-K** [gnulabis/UTFT-ESP8266#27]
I have a S6D1121_8 that works fine with Arduino and I would like to hook it up with ESP8266. I have tested ESP8266 there are 11 output available...

So in theory, I could use D0-D7 for the 8-bit interface, and D8, D9 (Rx) and D10 (Tx) for RS, WR and CS pin. For reset, just hope power-on reset will work (or maybe tie to the RST pin on ESP8266-12?)

**Comment by gnulabis** [comment]
Hi, as a quick first check, is this screen supported by the upstream UTFT?

**Comment by Derek-K** [comment]
Yes,  it does. I have tested with Arduino UNO using S6D1121_8 TFT driver.
